0.5/elements/feature-carousel.html

```diff
@@ -28,12 +28,18 @@
       interval: null, // num milliseconds to cycle panels.
       selected: 0,
       domReady: function() {
-        // TODO: use onMutation to watch for children updates.
         this.panels = Array.prototype.slice.call(
```

Comment:
Why don't we just do:

    this.childrenUpdated();

That sets `this.panels` and the mutation observer